programs/nodeos/logging.json

```diff
@@ -73,6 +82,15 @@
         "stderr",
         "net"
       ]
+    },{
```

Comment:
This wasn't part of another PR?

Reply:
No, it was missed on the original addition of this example logging.json